### DISCUSSION TITLE
Backup Einspielen RaspberryMatic auf HA OS nicht möglich

### DISCUSSION THREAD

**Discussion by horstvogel** [OpenCCU/OpenCCU#2774] [🇺🇸 General discussions]
Moin, 

ich kann beim Addon bei RaspberryMatic kein Backup einspielen.

Backup erstellen geht, Datei kann ich auch auswählen, wird dann angezeigt, aber wenn ich auf Backup einspielen drücke, dann passiert nichts. Im Screenshot ist das ohne Datei ausgewählt dargestellt,  Muss dann ein extra Fenster aufgehen?

![image](https://github.com/jens-maus/RaspberryMatic/assets/25108737/9e4a215d-ffbf-42c9-9d0a-9994bf0c6cf8)

**Comment by horstvogel** [comment]
Ich habe eine Neuinstallation ausgeführt, dann das Backup erstellt, auch dort ist es dann nicht möglich, dass man ein Backup einspielt. 
Gleiches habe ich mit dem Daily Snapsshot unter HA gemacht, wenn man auf Backup einspielen drückt, dann passiert nichts.

**Comment by Baxxy13** [comment]
Guck mal ob es in der Browserkonsole (F12 drücken) irgendwas auffälliges gibt.
Gehst du direkt über Ingress auf die WebUI?
Dann könntest du mal den externen Zugriff einrichten und es damit versuchen.
